bin/studio-common

```diff
@@ -75,7 +75,7 @@ if $is_newer; then
   echo "WARNING: Version '$installed_version' of Habitat has not been fully tested for compatibility with ci-studio-common.";
   echo "         Please consider exiting the sudio, running:";
```

Comment:
😄 While we're at it, could you fix that `sudio`, too, please?

Reply:
![](https://memegenerator.net/img/instances/48489683/sussudio-su-su-sussudio.jpg)